telemetry/definitions/commonDefinitions.json

```diff
@@ -773,6 +773,61 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "profileNameStatus",
+            "type": "string",
+            "description": "Shows the status of Profile Name input validations under IAM Credentials",
+            "allowedValues":[
+                "empty", "valid", "invalid"
+            ]
+        },
+        {
+            "name": "accessKeyStatus",
+            "type": "string",
+            "description": "Shows the status of AccessKey input validations under IAM Credentials",
+            "allowedValues":[
+                "empty", "valid", "invalid"
+            ]
+        },
+        {
+            "name": "secretStatus",
+            "type": "string",
+            "description": "Shows the status of Secret Key input validations under IAM Credentials",
+            "allowedValues":[
+                "empty", "valid", "invalid"
+            ]
+        },
+        {
+            "name": "numberOfConnections",
+            "type": "int",
+            "description": "Number of Existing connections the user has."
+            
+        },
+        {
+            "name": "builderIDStatus",
+            "type": "string",
+            "description": "If the user if successfully logged in",
+            "allowedValues":[ "connected", "non-connected"
+            ]
+            
+        },
+        {
+            "name": "iamIdentityCenterStartURLStatus",
+            "type": "string",
+            "description": "Shows the status of Start URL input validations under IAM Identity Center",
+            "allowedValues":[ "empty", "valid", "invalid" ]
+            
+        },
+        {
+            "name": "iamIdentityCenterRegion",
+            "type": "string",
+            "description": "Shows region selected by user under IAM Identity Center" 
+        },
+        {
+            "name": "errorMsg",
+            "type": "string",
+            "description": "Error message" 
```

Comment:
Where is this error message coming from? Should it be user-facing? Arbitrary errors can contain PII and should not be sent to telemetry.

Reply:
Updated the doc. Also we are using existing type : reason for this.